linchpin.py

```diff
@@ -114,3 +263,73 @@ def get(config, topo, layout):
     if layout:
         #click.echo("list called with layouts")
         get_file(config.clipath+"/inventory_layouts/"+layout,"./layouts/")
+
+@cli.command()
+@click.option("--lpf", default=False, required=False,  help="gets the topology by name")
+@pass_config
+def rise(config, lpf):
+    """ rise module of linchpin cli : still need to fix the linchpin_config and outputs, inventory_outputs paths"""
+    config.variable_manager.extra_vars = {}
+    init_dir = os.getcwd()
+    lpfs = list_by_ext(init_dir,".lpf")
+    if len(lpfs) == 0:
+        display("ERROR:001")
+    if len(lpfs) > 1:
+        display("ERROR:002") 
+    lpf = lpfs[0]
+    lpf = parse_yaml(lpf)
+    e_vars_grp = get_evars(lpf)
+    for e_vars in e_vars_grp:
+        """ need to change them to be a part of config obj"""
+        e_vars['linchpin_config'] = "/etc/linchpin/linchpin_config.yml"
```

Comment:
Will be changing the script according to PEP8.
Here on before PR is raised, will run through PEP8 checks